FogCreek/Glitch-Community
# Honored innocent: componentize layout

## Links
* Remix link: https://honored-innocent.glitch.me

## Changes:
* componentize Layout
* inline Konami redirect into Layout

## How To Test:
* visit home, user, team, project pages on https://honored-innocent.glitch.me and verify that the page layout isn't totally broken
* keying in the konami code `up up down down left right left right B A` should redirect you to https://honored-innocent.glitch.me/secret
